games-fps/prboom-plus/prboom_plus-2.5.1.4.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Doom source port"
+DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom project by Andrey Budko"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+COPYRIGHT="Andrey Budko 2016"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/prboom-plus/prboom-plus/$portVersion/prboom-plus-$portVersion.tar.gz"
+CHECKSUM_SHA256="e0e2241d174839b107f1a42b191aa9895dc6749e477cbc850ad520fba2353b94"
+SOURCE_DIR="prboom-plus-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Except for gcc2, there is no need for a secondary architecture package. `commandBinDir` is employed here to get rid of the unnecessary `secondaryArchSuffix` in binary names.

```diff
-SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir="$binDir"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+	SECONDARY_ARCHITECTURES="x86"
+	commandBinDir="$prefix/bin"
+fi
```